packages/xstate-wallet/src/store/indexedDB-backend.ts

```diff
@@ -17,7 +17,7 @@ enum ObjectStores {
 
 // A running, functioning example can be seen and played with here: https://codesandbox.io/s/elastic-kare-m1jp8
 export class IndexedDBBackend implements DBBackend {
-  private _db: any;
+  private _db: IDBDatabase;
```

Comment:
Where is this type defined?

Reply:
I would guess in the indexedDB type package?